docker-with-slurm/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM centos:7 AS slurm-build
+RUN yum -y install epel-release && yum clean all && rm -rf /var/cache/yum/*
+RUN yum -y install munge munge-devel libcgroup-devel curl bzip2 @buildsys-build \
+  readline-devel numactl-devel pam-devel glib2-devel hwloc-devel openssl-devel curl-devel \
+  && yum clean all && rm -rf /var/cache/yum/*
+COPY munge.key /etc/munge/munge.key
+RUN chown munge:munge /etc/munge/munge.key
+RUN chmod 0600 /etc/munge/munge.key
+RUN curl -o /tmp/slurm-19.05.4.tar.bz2 https://download.schedmd.com/slurm/slurm-19.05.4.tar.bz2
+RUN cd /tmp && tar xf slurm-19.05.4.tar.bz2
+RUN cd /tmp/slurm-19.05.4 && ./configure --prefix=/opt/slurm --sysconfdir=/opt/slurm/etc
+RUN cd /tmp/slurm-19.05.4 && make -j4
+RUN cd /tmp/slurm-19.05.4 && make install
+RUN mkdir /opt/slurm/etc
+RUN install -D -m644 /tmp/slurm-19.05.4/etc/slurmctld.service /opt/slurm/etc/
+RUN install -D -m644 /tmp/slurm-19.05.4/etc/slurmd.service /opt/slurm/etc/
```

Comment:
Don't we want all these RUN commands to be 1 item for layering?

Reply:
Yep I made that change. I initially had a bunch of `RUN` statements so speed up re-builds while getting everything working.